src/@types/spid-passport.d.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
@Undermaken we should move the type defs to the `spid-passport` package or else they get duplicated here and in `io-backend`